sale_order_revision/__init__.py

```diff
@@ -0,0 +1,9 @@
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+
```

Comment:
Please add `from . import models` in this file.

Reply:
Thank you for your review. Fixed.